src/Symfony/Component/DependencyInjection/Container.php

```diff
@@ -306,8 +306,7 @@ public function get($id, $invalidBehavior = self::EXCEPTION_ON_INVALID_REFERENCE
             try {
                 $service = $this->$method();
             } catch (\Exception $e) {
-                unset($this->loading[$id]);
-                unset($this->services[$id]);
+                unset($this->loading[$id], $this->services[$id]);
```

Comment:
ah, cool, I remember it was refused once.
in that case please adjust that one missing case inito:
```php
unset(
    $foo,
    $bar
);
```

and add `combine_consecutive_unsets` into `.php_cs.dist` to prevent creating consecutive `unsets` calls in the future